code/system/application/knowledge/include/linguistics/v2vcam/v2vcam.hpp

```diff
@@ -82,9 +82,14 @@ class V2vCam
   int32_t GetYawRateConfidence() const;
   int32_t GetVehicleRole() const;
 
+  bool m_initialised;
   std::ofstream m_sendLog;
   std::ofstream m_receiveLog;
   std::time_t m_timeType2004;
+  bool m_printInbound;
```

Comment:
Should we really have a separate way of printing data? Should we not instead use the OpenDaVINCI logging feature (if we need real-time), or simply rely on message logs?